airbyte-workers/src/main/java/io/airbyte/workers/config/ContainerOrchestratorConfigBeanFactory.java

```diff
@@ -76,6 +76,7 @@ public ContainerOrchestratorConfig kubernetesContainerOrchestratorConfig(
     environmentVariables.put(DD_DOGSTATSD_PORT_ENV_VAR, dataDogStatsdPort);
     environmentVariables.put(PUBLISH_METRICS_ENV_VAR, shouldPublishMetrics);
     environmentVariables.put(EnvVariableFeatureFlags.USE_STREAM_CAPABLE_STATE, Boolean.toString(featureFlags.useStreamCapableState()));
+    environmentVariables.put(EnvVariableFeatureFlags.AUTO_DETECT_SCHEMA, Boolean.toString(featureFlags.autoDetectSchema()));
```

Comment:
I think so? I was copying how we were doing it for the per stream state env variable

Reply:
I think for stream state, it was added to the orchestrator because it was needed to know which behavior to pick when sending a state to the destination.
For auto detect schema, I expect the changes should impact the server, the worker if we want to have flag to control whether we register the new workflow. I don't think it impacts replication so it should not be needed in the orchestrator.